Jenkinsfile_CNP

```diff
@@ -113,23 +106,20 @@ withPipeline(type, product, component) {
 
     // Check if the build should be wired to an environment higher than 'preview'.
     if (env.BRANCH_NAME == 'master' || env.BRANCH_NAME == 'demo' || env.BRANCH_NAME == 'perftest' || env.BRANCH_NAME == 'ithc' || prsToUseAat.toLowerCase().contains(env.BRANCH_NAME.toLowerCase())) {
-        environmentOfDependencies = env.BRANCH_NAME
-        // use aat environment for dependencies of master and CI/CD builds.
-        if (env.BRANCH_NAME.startsWith("PR") || env.BRANCH_NAME == 'master') {
-            environmentOfDependencies = "aat"
-        }
-        env.DEFINITION_STORE_HOST      = "http://ccd-definition-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
         env.ELASTIC_SEARCH_FTA_ENABLED = "true"
+    } else {
+        env.DEFINITION_STORE_URL_BASE = "http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal"
     }
 
     echo "ES FTA Enabled = ${env.ELASTIC_SEARCH_FTA_ENABLED} on branch ${env.BRANCH_NAME}"
-    echo "FTA will use Definition Store: ${env.DEFINITION_STORE_HOST}"
+    echo "FTA will use Definition Store: ${env.DEFINITION_STORE_URL_BASE}"
```

Comment:
please remove this line. will print null many time.

Reply:
done